### PR TITLE
[skip ci] PullApprove

### PR DESCRIPTION
**Pull request by gigawhitlocks**
Please pardon all my testing commits at the top. They'll get squashed.
Refer to https://gist.github.com/gigawhitlocks/5d7f0333e1336f3928fdc1215d90ba82 for what this file represents with regard to component owners etc.

You can also look at the yml file and do ctrl+f and search for your github username and look at the paths assigned to you. 

Differences from the above: I didn't assign @hickeng to anything that's labelled "george?" because I dont want to default to him since he has so much on his plate. Also I did not add the UI guys as component owners for the UI bits, as I'm not sure whether I should be adding @jooskim or @jak-atx

### CODE REVIEW

**Comment by gigawhitlocks** [comment]
LGTM

**Comment by gigawhitlocks** [comment]
Approve

**Comment by gigawhitlocks** [comment]
Approved

**Comment by jooskim** [comment]
Hey Ian,
Owner for `cmd/vic-ui` should be @andrewtchin, as he wrote `vic-ui-*`. Also you could put me (jooskim) for ui/* as I've been in charge of the actual development for the past few months. Josh Kenney is still engaged in VIC but has more focus on the other project right now. Thanks for the heads-up

**Comment by gigawhitlocks** [comment]
LGTM

File: .pullapprove.yml [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  port-layer-network:
+    users:
+      - hmahmood
+    conditions:
+      files:
+        includes:
+          - "cmd/vic-dns/*"
+          - "lib/dns/*"
+          - "lib/dhcp/*"
+          - "lib/portalyer/network/*"
```
Comment (anchal-agrawal) [inline .pullapprove.yml]: Typo in portlayer.

**Comment by gigawhitlocks** [comment]
Approved

**Comment by emlin** [comment]
Just one general question for it. What if we added new directories, or just rename or move to somewhere else?

**Comment by gigawhitlocks** [comment]
@emlin if that happens, the yml file will have to be updated to reflect the proper owner of the file, if it's moved out of your tree. The `git mv` action should cause the move to need to be approved by the current owner of the file.

**Comment by sflxn** [comment]
The persona also includes vic/cmd/docker

**Comment by dougm** [comment]
@gigawhitlocks vcsim is mainly under pkg/vsphere/simulator

toolbox is not part of vcsim (its our guest tools impl), most of toolbox lives under pkg/vsphere/toolbox

to summarize:
vcsim: "cmd/vcsim/*" "pkg/vsphere/simulator/*"
toolbox: "cmd/toolbox/*" "pkg/vsphere/toolbox/*"

otherwise lgtm

**Comment by gigawhitlocks** [comment]
Thanks @dougm and @sflxn 
@dougm , @fdawg4l told me to assign toolbox to you but I will make the adjustments as advised

**Comment by gigawhitlocks** [comment]
Approved

**Pull request merged**